src/components/shared/testimonials-card.json

```diff
@@ -0,0 +1,43 @@
+{
+  "collectionName": "components_shared_testimonials_cards",
+  "info": {
+    "displayName": "TestimonialCard",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "icon": {
+      "type": "media",
+      "multiple": true,
+      "required": false,
+      "allowedTypes": [
+        "images"
+      ]
+    },
+    "title": {
+      "type": "string",
+      "maxLength": 256
+    },
+    "text": {
+      "type": "text",
+      "required": true
+    },
+    "author": {
+      "type": "text"
+    },
+    "bgColor": {
```

Comment:
Suggestion: Let's have a theme here instead to have the ability to change text color as well.

Reply:
I can set theme for all component Testimonials, I can’t do it separately for cards

Reply:
Yes, I think so.

Reply:
According to the design they might have different colors. Let's clarify with Iryna if it's ok to have a theme for all cards, or not. 
<img width="831" alt="image" src="https://github.com/user-attachments/assets/770c402e-9075-4d57-bbd8-c3b21f68e74a">

I think it's possible to do it for the card if we move the theme to the collection types.

<img width="218" alt="image" src="https://github.com/user-attachments/assets/2e239c33-0d54-4362-9981-7b1c89f93902">

Reply:
In case of selecting cild components I able to chose only "shared",  "section" and "metadata". 
I can't select any part of collection types. 
Let's discuss it on a sync if there ways to use it from "collection types"

Reply:
> Suggestion: Let's have a theme here instead to have the ability to change text color as well.

I've discussed with Iryna to add theme for Testimonials and TestimonialsCards, and make it as relation. 
The changes has been applied